examples/joy_circle_grid.ml

```diff
@@ -0,0 +1,30 @@
+open Joy.Shape
```

Comment:
Please remove this file and the dune change. I believe they're from a different PR.

Reply:
Sure. Will do

Reply:
Hello @Sudha247 I have made changes, can you kindly help me review them. Thanks a lot